src/index.html

```diff
@@ -20,8 +21,530 @@
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-cost">$2,199</div>
+        </div>
+
+        <button
+          type="button"
+          data-qa="card-hover"
+          class="card__button"
+        >
+          BUY
+        </button>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image-container">
+          <img
+            src="./images/imac.jpeg"
+            alt="item card"
+            class="card__image"
+          />
+        </div>
+
+        <div class="card__name">
+          <div class="card__name-main">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__name-product">Product code: 195434</div>
+        </div>
+
+        <div class="card__review">
+          <div
+            class="card__review-stars"
+            aria-label="four stars"
+          >
+            <div class="card__review-stars-star"></div>
+
+            <div class="card__review-stars-star"></div>
+
+            <div class="card__review-stars-star"></div>
+
+            <div class="card__review-stars-star"></div>
+
+            <div class="card__review-stars-star"></div>
+          </div>
+
+          <div class="card__review-review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-cost">$2,199</div>
+        </div>
+
+        <button
+          type="button"
+          data-qa="hover"
+          class="card__button"
+        >
+          BUY
+        </button>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image-container">
+          <img
+            src="./images/imac.jpeg"
+            alt="item card"
+            class="card__image"
+          />
+        </div>
+
+        <div class="card__name">
+          <div class="card__name-main">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__name-product">Product code: 195434</div>
+        </div>
+
+        <div class="card__review">
+          <div
+            class="card__review-stars"
+            aria-label="four stars"
+          >
+            <div class="card__review-stars-star"></div>
+
+            <div class="card__review-stars-star"></div>
+
+            <div class="card__review-stars-star"></div>
+
+            <div class="card__review-stars-star"></div>
+
+            <div class="card__review-stars-star"></div>
+          </div>
+
+          <div class="card__review-review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-cost">$2,199</div>
+        </div>
+
+        <button
+          type="button"
+          data-qa="hover"
```

Comment:
The data-qa attribute value is inconsistent with other similar elements. Consider using a consistent naming convention for data attributes.